app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative '../services/evaluator_management_service'
```

Comment:
is this line needed? I was under the impression that Rails handles requiring all the code under `app/`?

Reply:
still works without it 👍